src/components/Model3D.tsx

```diff
@@ -136,71 +162,67 @@ class Model3D extends React.Component {
       this.selectNode(nodeId, true, 0);
     }
 
-    this.model._treeIndexNodeIdMap.forEach(id => {
-      this.model.setNodeColor(id, 170, 170, 170);
+    // TODO, this is not valid...
+    // @ts-ignore
+    this.model!._treeIndexNodeIdMap.forEach(id => {
+      this.model!.setNodeColor(id, 170, 170, 170);
     });
   };
 
-  onClick = nodeId => {
+  onClick = (nodeId: number) => {
     this.setState({ nodeId });
   };
 
-  onReady = (viewer, model) => {
+  onReady = (viewer: Cognite3DViewer, model: Cognite3DModel) => {
     this.viewer = viewer;
     this.model = model;
     // Temp disable screen space culling
+    // TODO, this is not allowed
+    // @ts-ignore
     model._screenRatioLimit = 0.0;
```

Comment:
Why is this here in the first place?

Reply:
I think i'll figure this out later, it so far seems to have no effect